apps/commercetools/src/Editor/CategoryPreviews/SortableListItem.tsx

```diff
@@ -71,48 +76,47 @@ const styles = {
 };
 
 const CardDragHandle = SortableHandle(() => (
-  <FormaCardDragHandle className={styles.dragHandle}>
-    Reorder category
-  </FormaCardDragHandle>
+  <FormaCardDragHandle className={styles.dragHandle}>Reorder category</FormaCardDragHandle>
 ));
 
+function getCategoryIdentifier(category: Category) {
+  return category.slug.length ? category.slug : category.id;
+}
+
 export const SortableListItem = SortableElement<Props>(
   ({ category, disabled, isSortable, onDelete }: Props) => {
-    const categoryIsMissing = !category.name;
-
     return (
       <Card className={styles.card}>
         <>
           {isSortable && <CardDragHandle />}
           <section className={styles.description}>
             <Typography>
-              <Heading className={styles.name(category.name)}>
-                {categoryIsMissing ? category.id : category.name}
+              <Heading className={styles.heading(category)}>
+                {category.isMissing || !category.name
```

Comment:
Optional: you use the same bool logic here as you do when you pass it into the `styles.heading` func. You could have that logic in var and reuse it.